### ISSUE TITLE
[FEATURE] Implement clean way for fetching additional fields, like description_frontend

### ISSUE THREAD

**Issue by neufeind** [Kephson/reint_downloadmanager#19]
Currently description_frontend is "uncleanly" fetched from the database and setDescription used to override the existing (but not used) field "description" in the domain-model. Trying to provide additional fields like "description2" (in my case) that does not work, since there is no clean way to fetch the additional fields from the database and there is no setDescription2() in the model.
But with EXT:extender it would be possible to cleanly add "description_frontend" to the model (with getter/setter). This way, in your own extension providing another field, it would be possible to also provide additional fields. Then through some kind of configuration EXT:reint_downloadmanger could be told to fetch "description_frontend" and possible other fields from the database and to inject them through proper set-calls into the (extended) model.

Basically I'm trying to find a cleaner way to provide "description_frontend" inside reint_downloadmanager without "abusing" the internally already provided setDescription() - and have that same approach cleanly for adding even more fields by simply creating a small extension providing them (tca, calls to EXT:extender and typoscript maybe).

**Comment by Kephson** [comment]
Hello @neufeind I will check this.

**Comment by Kephson** [comment]
This could be done e.g. like this: https://docs.typo3.org/p/in2code/femanager/main/en-us/Features/NewFields/Index.html
